docs/topics/downloader-middleware.rst

```diff
@@ -1249,8 +1249,7 @@ AJAXCRAWL_ENABLED
 
 Default: ``False``
 
-Whether the AjaxCrawlMiddleware will be enabled. You may want to
-enable it for :ref:`broad crawls <topics-broad-crawls>`.
+Whether the AjaxCrawlMiddleware will be enabled.
```

Comment:
It probably makes sense to add a note below "AjaxCrawlMiddleware Settings", explaining that this middleware is deprecated.